packages/create-miniapp-cli/template/server/index.js

```
@@ -0,0 +1,25 @@
+const { prepareKeystone } = require('@open-condo/keystone/KSv5v6/v5/prepareKeystone')
+
+const { CondoOIDCMiddleware } = require('./middlewares/oidc')
+
+const apps = () => [
+    FileAdapter.makeFileAdapterMiddleware(),
+    new HealthCheck({ checks: [getPostgresHealthCheck(), getRedisHealthCheck()] }),
+    new CondoOIDCMiddleware(),
+]
+
+const schemas = () => [
+    require('~/domains/user/schema'),
```

ESLint

[error] '~/domains/user/schema' module is restricted from being used by a pattern.

(no-restricted-modules)